proposals/umoci.md

```diff
@@ -0,0 +1,188 @@
+# OCI umoci Project Proposal #
+
+## Abstract ##
+The need for a "works out of the box" image tool that is supported by OCI has been clear for several years.
+umoci was initially developed to fulfil this requirement, and after several years of development and wide-spread production usage, we feel it is time to include it within the OCI.
+The following proposal outlines how this would be achieved.
+
+## Background ##
+To quote the description of umoci from [the project website][umoci]:
+
+> umoci is a free software tool for manipulating and interacting with container
+> images in the standardised Open Container Initiative’s image format. It
+> provides one of the most flexible image management toolsets, requiring
+> neither a daemon nor any particular filesystem setup. It is already used in a
+> variety of different projects and by several companies.
+
+umoci is primarily used as a command-line tool, and can be used to perform fundamental operations on an OCI container image.
+An example session looks like
+
+```shell
+$ umoci unpack --image opensuse:leap bundle
+$ runc run -b bundle ctr
+# # make some changes...
+$ umoci repack --image opensuse:leap bundle
+```
+
+Where `opensuse` is an OCI image layout stored as a directory, and `leap` is a "tag" in the OCI image.
+The above session extracted (`umoci unpack`) the image as an OCI OCI Runtime Specification bundle, spawned a container using the bundle (`runc run ...`), and then updated the image to create a new image layer based on any changes made (`umoci repack`).
```

Comment:
s/OCI OCI/OCI/
a few instances...

Reply:
Oops, that's search-replace for you (I replaced all `image-spec` instances with `OCI Image Specification`).